demos/background_subtraction_demo/python/background_subtraction_demo.py

```diff
@@ -68,6 +68,10 @@ def build_argparser():
                            'same shape as an input image.')
     args.add_argument('--blur_bgr', default=0, type=int,
                       help='Optional. Background blur strength (by default with value 0 is not applied).')
+    args.add_argument('--layout', type=str, default=None,
+                      help='Optional. Model inputs layouts. '
+                           'Format "NCHW" or "<input1>:<layout1>,<input2>:<layout2>" in case of more than one input.'
```

Comment:
Input tensor names might have `:`, so delimiter should be changed https://github.com/openvinotoolkit/open_model_zoo/pull/3104#discussion_r804716826